.github/workflows/release-asset-publish.yml

```diff
@@ -0,0 +1,42 @@
+name: Publish release .jar
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    name: Publish release .jar
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ 17 ]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          cache: 'maven'
+          java-version: ${{ matrix.java }}
+
+      - name: Build base .jar via Maven
+        run: mvn clean package -Dcheckstyle.skip -Dmaven.test.skip=true
+      - name: Upload and attach base .jar to release
+        uses: svenstaro/upload-release-action@v2
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: target/prebid-server.jar
+          asset_name: prebid-server-${{ github.ref_name }}.jar
+          tag: ${{ github.ref }}
+
+      - name: Build bundled .jar via Maven
+        run: mvn clean package --file extra/pom.xml -Dcheckstyle.skip -Dmaven.test.skip=true
+      - name: Upload and attach bundled .jar to release
+        uses: svenstaro/upload-release-action@v2
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: extra/bundle/target/prebid-server-bundle.jar
+          asset_name: prebid-server-bundle-${{ github.ref_name }}.jar
+          tag: ${{ github.ref }}
```

Comment:
since tag version is set up manually, we might need to define the overwrite strategy in case of the duplicates

Reply:
Yes, this action will overwrite exiting asset, if github.ref_name will point to the same release (that is quite problematic, if you will look into release drafter action). But AFAIK, duplicated release tags - it's not ok at all, actually. And such case should not be allowed by GitHub itself.

Reply:
From the description of the action you use by default overwriting is disabled. 

I don't like having this action to be dependent on the release drafter, because it's a separate post-step. So it's better to consider cases when "oops happens" and we might get outdated artifacts or artifacts that doesn't match the actual release codebase and we need to remove/overwrite them. I mean doing it manually will be painfull.

Reply:
I believe the person who usually releases can judge and say what kind of problems can be and how they are usually mitigated. Overall the action will work as intended, and 99% of releases will be smooth.